### ISSUE TITLE
오류의 내용

### ISSUE THREAD

**Issue by kevinhojae** [binsoopark/FlutterProjectExamples#1]

**Comment by binsoopark** [comment]
오류의 내용이 어떤 것일까요?

**Comment by ingkkukiii** [comment]
Error: The getter 'addressOf' isn't defined for the class 'GUID'.
 - 'GUID' is from 'package:win32/src/structs.dart' ('../../../../../../flutter/.pub-cache/hosted/pub.dartlang.org/win32-1.7.4/lib/src/structs.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'addressOf'.
          knownFolderID.addressOf, KF_FLAG_DEFAULT, NULL, pathPtrPtr);
                        ^^^^^^^^^
(생략)
...
3장 소스코드를 돌려보면 콘솔창에 다음과 같은 에러가 뜨네요..? 원인을 알 수 있을까요?

**Comment by binsoopark** [comment]
@ingkkukiii 님,
Github의 프로젝트 소스를 Clone 하신 상태에서, 별도의 추가 작업 없이, ch_03 폴더 내의 productivity_timer 폴더를 Android Studio에서 열고 실행하신 상태이신가요?
아니면, 책에 있는 내용을 따라 진행하시다가 특정 부분에서 발생하시는 문제일까요?

현 시점에서 확인해보았을 때에는 AsyncTask() 관련 warning이 7개 정도 발생하기는 하지만 여전히 실행에는 문제가 없는 것으로 확인이 됩니다.

혹시 그리고 여전히 문제가 발생하신다면 확인된 flutter 버전이 어떤 버전인지 말씀 부탁드립니다.😄 
저는 `Channel stable, 2.0.1` 기준으로 확인하였습니다.

**Comment by ingkkukiii** [comment]
네 안녕하세요! 우선 메일 주셔서 감사합니다
 
저의 현재 플러터 다트 버전입니다.
말씀하신 대로 별도의 추가 작업 없이 깃소스를 다운받아 실행했는데 아래와 같은 오류가 발생하는 상황입니다
 
 
 
혹시 해결 방안이나 안내 사항을 제시해 주실 수 있으시면 메일 부탁드리겠습니다.
감사합니다.
-----Original Message-----

**Comment by binsoopark** [comment]
네, 환경이 어떻냐에 따라서 해결방안이 달라질 수 있어서요. 소스를 그대로 가져와서 실행하셨다고 한다면, 현재 설치되어 있는 플러터 및 다트 버전을 알고 싶습니다. 또한 에러메시지도 될 수 있는대로 전부 다 알고싶은데요. 혹시 이곳에 로그를 텍스트 파일로 올려주실 수 있으신가요?

**Comment by ingkkukiii** [comment]
답변이 늦었습니다 이렇게 올리면 되는건가요???
[error.txt](https://github.com/binsoopark/FlutterProjectExamples/files/7323929/error.txt)
[version.txt](https://github.com/binsoopark/FlutterProjectExamples/files/7323930/version.txt)

**Comment by binsoopark** [comment]
음.. 어디서부터인지 종속성 문제가 있는것 같은데요. 시도할 수 있는 방법은 아래 두 가지 중 하나일듯싶습니다.

### 1
첫 번째 방법으로는
-  .idea 폴더를 삭제
- build 폴더를 삭제
- pubspec.lock 파일을 삭제
![image](https://user-images.githubusercontent.com/6077774/137357529-0289b93d-4ee5-49d0-bfc4-708464205c94.png)
- 터미널이나 커맨드 창에서 프로젝트 루트로 이동 후 `flutter pub get` 명령 실행

### 2
만약 위의 방법으로 해결이 안 된다면 두번째로는 비슷한 방식인데 다음의 명령어를 프로젝트 루트의 터미널이나 커맨드 창에서 실행해보세요.
- `flutter clean`
- `flutter pub get`
- `flutter pub upgrade`


기존에 설치된 종속성을 깔끔하게 정리하고 다시 가져오는 방법입니다.
위와 같이 시도 후 다시 Run을 실행해보세요~
